From 8e254c4c92c3d09dcfef1ba98b7023bea246adeb Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Thu, 3 May 2012 09:24:32 +1000
Subject: [PATCH] data: add generic eraser to Bamboo Pen & Touch

Crashes gnome-control-center without it.

(gnome-control-center:18115): wacom-cc-panel-WARNING **: Could not set the
current stylus ID 0x0 for tablet 'Wacom Bamboo Pen & Touch', no general pen
found
wacom-cc-panel:ERROR:gsd-wacom-device.c:1491:gsd_wacom_device_set_current_stylus:
assertion failed: (device->priv->styli)

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 data/bamboo-pen-and-touch.tablet |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/bamboo-pen-and-touch.tablet b/data/bamboo-pen-and-touch.tablet
index 125db37..3fff338 100644
--- a/data/bamboo-pen-and-touch.tablet
+++ b/data/bamboo-pen-and-touch.tablet
@@ -13,7 +13,7 @@ Width=6
 Height=4
 
 # The other stylusses don't match it, it has two buttons and an eraser.
-Styli=0xfffff
+Styli=0xfffff;0xffffe;
 
 [Features]
 Stylus=true
-- 
1.7.10

